skgstat/Variogram.py

```diff
@@ -996,22 +986,41 @@ def fitted_model(self):
         # get the pars
         cof = self.cof
 
-        # get the function
-        func = self._model
-
-        if self._harmonize:
-            code = """model = lambda x: func(x)"""
+        return self.fitted_model_function(self._model, self.cof)
+
+    @classmethod
+    def fitted_model_function(cls, model, cof=None, **kw):
+        if cof is None:
+            # Make sure to keep this synchronized with the output
+            # of describe()!
+            cof = [kw["effective_range"], kw["sill"]]
+            if "smoothness" in kw:
+                cof.append(kw["smoothness"])
+            if "shape" in kw:
+                cof.append(kw["shape"])
+            if kw["nugget"] != 0:
+                cof.append(kw["nugget"])
+
+        harmonize = False
+        if not callable(model):
+            if model == "harmonize":
+                # FIXME: How do we serialize a harmonized model in describe()?
```

Comment:
This is where I don't really know what to do...

Reply:
I will have a look at it as soon as possible.

Reply:
Hey @redhog, I don't find the time right now to solve this (lockdown here, kids at home etc.)...
Would it be a possible pathway for you to **not** include the parameters of harmonized models for now? Then, the Kriging class would raise a `NotImplementedError` if model params are missing. Then, we could merge this PR and open an issue concerning the harmonized model, to keep it on the agenda.
I just don't want to hold you up any longer, sry about that.

Reply:
I don't need harmonized models myself, so for me that'd work. However, it would break the workflow for anyone who do. If you think that's not too big a deal, we could merge anyway. Alternatively I guess I could add a hack workaround: Copy the model function (not just name) here: https://github.com/mmaelicke/scikit-gstat/blob/cf0751b5c45b9acaacb7cb2086c42fa881d440a7/skgstat/Kriging.py#L105

Reply:
Then I would go for hacky workaround along with an issue opened to fix it at any later point in time.